src/extensions/yfm/YfmCut/YfmCutSpecs/index.ts

```diff
@@ -12,9 +12,9 @@ import {toYfm} from './toYfm';
 export {CutNode, cutType, cutTitleType, cutContentType} from './const';
 
 export type YfmCutSpecsOptions = {
```

Comment:
I suggest renaming the components as well, although it might be better to do that next.

Reply:
Yeah, there will be more renaming in future PRs.
But we will not rename these components, because it's part of the YFM syntax